Landry333/Big-Owl
# #202 Fixed Missing Notifications

### Related Issues
Closes #202 

### Extra Note
I actually made an improvement to the way Notification are set so that when we code new Notifications which can be seen in this commit (https://github.com/Landry333/Big-Owl/commit/dd56126498aa4275459570384db574e6be0367ee). This is because there is less stuff for us to maintain and worry about But it would effect the type field of the Notifications (which would make the current Notifs in our db invalid), and it would require a slight amount of changes to the code for Notification page (although it should be relatively simple)

### Description
I fixed the bug where the ScheduleRequests would not show up in the notification list and made some changes to prevent us from causing that to happen again

### How to check PR
- Create a Schedule for a user (ex: send a schedule to TestUser2 using TestUser1)
- Login as the User you sent a Schedule to (ex: TestUser2) and accept or reject this schedule
- Find the notification created as a result of accepting/rejecting a schedule and the type field should match the Constant here (https://github.com/Landry333/Big-Owl/blob/1c8f8cdea55677d6fcd4b04414f0914bf814b58e/app/src/main/java/com/example/bigowlapp/model/Notification.java#L12)
